README.md

```diff
@@ -3,35 +3,82 @@
 
-`$ npm i`
+`$ nvm use` # will install/setup node needed for this project
+`$ npm i`   # will install deps
 
 then, start the dev server:
 
 `$ npm start`
 
-### Where is it running?
-The project itself will run on http://localhost:8734
+Visit http://localhost:8734
 
-### Setup prerequisites for full development
+At this point, it will ask you to login. Use the user indicated in devstack (edx@example.com, edx). When logged in, expect to see a 404 since we are not setup
 
-To setup test data, and other backend, please follow these steps, you will setup a `test-enterprise` customer who will have learners associated with it:
+Now quickly check the .env.development file for which services are used by the application. e.g. `LMS_BASE_URL='http://localhost:18000'` . We will get back to this.
 
- - Start backend services needed (refer to .env.development file). Refer to [Devstack Doc](https://github.com/edx/devstack) for details
- - Seed test data. Refer to [The Enterprise management commands](https://github.com/edx/edx-enterprise/blob/master/enterprise/management/commands/seed_enterprise_devstack_data.py#L47). Usually this means running something like `./manage.py lms seed_enterprise_devstack_data`, but refer to the link for latest info
- - Ensure the Enterprise Integration flag is enabled in devstack [see this link](https://github.com/edx/edx-platform/blob/master/lms/envs/devstack.py#L326)
- - Restart any services such as `make lms-restart` for applying changes
- - Pick a user such as enterprise_learner_1 to use as learner account (password is the same as the test edx account documented on devstack)
- - Go to http://localhost:8734/test-enterprise/ to browse the application for the `test-enterprise` customer
+
+## Setup enterprise stack
+
+An enterprise portal will need a couple of roles: The Enterprise customer, and at least one learner account.
+We also need the edx-enterprise Django module to be running inside of the LMS (which is started by devstack). Refer to [Devstack Doc](https://github.com/edx/devstack) for details. This is what serves endpoints used by the application.
+
+* Setup devstack if not already and at least have the `lms` service running. Typically `make dev.up` or `make dev.nfs.up` will do it
+* Ensure you can browser to LMS at least, http://localhost:18000
+
+* Sync edx-enterprise into the src/ folder of your work folder (e.g. ~/work/src/edx-enterprise).
+* Ensure this env var is set in your env for the docker mount to be located correctly, e.g.,: `DEVSTACK_WORKSPACE=/Users/$USER/work` , in your shell config file
+* Either start new shell or run `exec "$SHELL"` for changes to take effect in the shell
+* Start backend services needed (refer to .env.development file). Refer to [Devstack Doc](https://github.com/edx/devstack) for details
+* Now in your devstack folder run
+  ```
+  $ make lms-shell
+  $ pip install -e /edx/src/edx-enterprise to uninstall the published edx-enterprise package and instead install your local copy from src.
```

Comment:
I saw the thread this bit came from and this is not a step for getting the stack setup.
This is saying "you've changed something in edx-enterprise and want to verify that change in the application that actually uses it as a library".
If you haven't changed edx-enterprise and just want to have the stack running, none of the copying enterprise, reinstalling it, etc. steps are needed here.
I was looking at actually add the steps to get edx-enterprise running in lms to test stuff into the edx-enterprise repo, but then it didn't seem like you can actually successfully run tests in LMS, so all that went away. 

But tl;dr this step isn't relevant to just having the apps run.